tests/k8s/helpers.bash

```diff
@@ -29,3 +29,17 @@ function abort {
 
 	exit 1
 }
+
+function gather_k8s_logs {
+  local LOCAL_NODE_NUM=$1
+  local LOGS_DIR=$2
+
+  mkdir -p ${LOGS_DIR}
+  local CILIUM_PODS=$(kubectl get pods -n ${NAMESPACE} -l k8s-app=cilium | tail -n +2 | awk '{print $1}')
```

Comment:
Can you retrieve the node's name as well? `kubectl get pods -o wide` gives you that info, it's just a matter of playing with `awk` to join both names together.